gustaf/utils/init_helper.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any
+
+
+class LibraryCanNotBeLoadedHelper():
+    """
+    Class used to have better import error handling in the case that a package
+    package is not installed. This is necessary due to that some packages are
+    not a dependency of `gustaf`, but some parts require them to function.
+    Examples are `splinepy` and `vedo`.
```

Comment:
```suggestion
    Class used to improve import error handling in the case some requested
    module or package is unavailable. Class replaces all instances of
    requested object and throws an error when accessed.

    This class enables `gustaf` to provide all supplementary features, even
    if specific features might not be supported.
    Examples are `splinepy` and `vedo`
```

Its just a suggestion, do as you like :smile: